test/internet/test-dns.js

```diff
@@ -129,6 +129,24 @@ TEST(function test_resolveSrv(done) {
   checkWrap(req);
 });
 
+TEST(function test_resolvePtr(done) {
+  var req = dns.resolvePtr('8.8.8.8.in-addr.arpa', function(err, result) {
+    if (err) throw err;
+
+    assert.ok(result.length > 0);
+
+    for (var i = 0; i < result.length; i++) {
+      var item = result[i];
+      assert.ok(item);
+      assert.ok(typeof item === 'string');
+    }
+
+    done();
+  });
+
+  checkWrap(req);
+});
+
```

Comment:
Oh, and possibly also one for `dns.resolvePtr('8.8.8.8')` which should result in an error.